src/components/JfDrawerComponent.spec.js

```diff
@@ -0,0 +1,28 @@
+import {shallowMount} from '@vue/test-utils';
+import JfDrawerComponent from './JfDrawerComponent.vue';
+
+describe('JfDrawerComponent', () => {
+    it('should set open variable to be true', () => {
+        const wrapper = shallowMount(JfDrawerComponent, {
+            propsData: {
+                openFirst: '0'
+            },
+            stubs: ['b-collapse', 'b-card']
+        });
+        expect(wrapper.vm.opened).toBe(true);
+    });
+});
+
+
+describe('JfDrawerComponent', () => {
+    it('should fireResizeEvent when opened', () => {
+        const wrapper = shallowMount(JfDrawerComponent, {
+            stubs: ['b-collapse', 'b-card']
+        });
+        let vm = wrapper.vm;
+        vm.utils = {};
+        vm.utils.fireResizeEvent = jest.fn();
+        vm.onClickHeader();
```

Comment:
IMHO you should click on the proper html and simulate full event handling pipe from v-on:click to vm.utils.fireResizeEvent.